ospf6d/ospf6_asbr.c

```diff
@@ -1645,8 +1645,13 @@ DEFUN (ospf6_redistribute,
 	red = ospf6_redist_lookup(ospf6, type, 0);
 	if (!red)
```

Comment:
Please, also add brackets to the `if` part. According to the coding style, if the brackets are used in any part of the if-else statement, they should be used in every part.